test-node.bash

```diff
@@ -80,21 +96,27 @@ while [[ $# -gt 0 ]]; do
             ;;
         --init-force)
             force_init=true
+            build_utils=true
+            build_node_images=true
             shift
             ;;
         --dev)
             simple=false
             shift
             if [[ $# -eq 0 || $1 == -* ]]; then
                 # If no argument after --dev, set both flags to true
-                dev_build_nitro=true
-                dev_build_blockscout=true
+                dev_nitro=true
+                build_dev_nitro=true
+                dev_blockscout=true
+                build_dev_blockscout=true
             else
                 while [[ $# -gt 0 && $1 != -* ]]; do
                     if [[ $1 == "nitro" ]]; then
-                        dev_build_nitro=true
```

Comment:
It seems like, in conjunction with naming nits, the upstream changes also remove the usage of dev_build_nitro/blockscout and don't remove the variables. Maybe we should just remove those variables to cut down on confusion?

Reply:
makes sense - https://github.com/EspressoSystems/nitro-testnode/pull/63/commits/8683c89a89e524a71b9de29a3dc82e30dffbc96d